changelog.md

```diff
@@ -7,6 +7,50 @@ All notable end-user facing changes are documented in this file.
 *Here are all the changes in `master` branch, and will be moved to the appropriate release once they are included in a published nuget package.
 The idea is to track end-user facing changes as they occur.*
 
+### [3.2.0-rc2] (changes since 3.2.0-rc1)
+
+- Non-breaking bug fixes
+  - Do not reject rejection messages locally. Drop them instead (#6525)
+
+### [3.1.7] (changes since 3.1.6)
+
+- Non-breaking bug fixes
+  - Do not reject rejection messages locally. Drop them instead (#6525)
+
+### [3.2.0-rc1] (changes since 3.1.6)
+
+- Breaking changes
+  - Remove current multicluster implementation (#6498)
+
+- Non-breaking improvements
+  - Support for pluggable grain directory (#6340, #6354, #6366, #6385, #6473, #6485, #6502, #6524)
+  - Expose timeouts for Azure Table Storage (#6462, #6501, #6509)
+  - Schedule Tasks and WorkItems on .NET ThreadPool (#6261)
+  - Schedule received messages onto thread pool in Connection.ProcessIncoming (#6263)
+  - Remove AsyncAgent, Executor and related (#6264)
+  - Reorient RuntimeContext around IGrainContext (#6365)
+  - Remove Message.DebugContext and related code (#6323)
+  - Remove obviated GrainId constructor and associated code (#6322)
+  - Set isolation level to `READ COMMITED` to avoid Gap Lock issues (#6331)
```

Comment:
typo in COMMITTED

Reply:
Yeah, this was a typo in the commit message. Fixed.